test/library/packages/Sort/RadixSort/simpleMerge.chpl

```diff
@@ -0,0 +1,53 @@
+proc merge(array:[] int, l: int, m: int, r:int){ 
+    
+    var i: int;
+    var j: int; 
+    var k: int; 
+    var n1 = m - l + 1; 
+    var n2 =  r - m; 
+  
+    var L:[1..n1] int;  
+    var R:[1..n2] int;
+
+    for i in 1..n1 { 
+        L[i] = array[l + i - 1]; 
+    }
+    for j in 1..n2 {
+        R[j] = array[m + j]; 
+    }
+
+    i = 1; 
```

Comment:
I would encourage declaring these variables here rather than at the top of the function, e.g.

```chpl
var i = 1,
    j = 1,
    l = l;
```